### ISSUE TITLE
Show projects with unavailable backends as disabled in the UI

### ISSUE THREAD

**Issue by r4victor** [dstackai/dstack#599]
Now, if the backend is not available, the project is not listed on the Projects page. The API does not include the project in the response. Instead, the project should be shown, but as disabled (so it can be deleted, for example).

**Comment by r4victor** [comment]
No longer relevant after #633
